index.d.ts

```diff
@@ -277,6 +277,10 @@ export interface AxiosProgressEvent {
 
 type Milliseconds = number;
 
+type AxiosAdapterName = 'xhr' | 'http' | string;
```

Comment:
why not `'xhr' | 'http'` ?
currently it does not provide auto completion in IDEs because of that `| string`